### PR TITLE
BXMSPROD-779 webhook configuration

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.jboss.org/browse/BXMSPROD-779

### CODE REVIEW

File: job-dsls/jobs/kie_jenkinsScripts_PR.groovy [hunk 1]
```diff
@@ -84,13 +84,14 @@ job(jobName) {
 
     triggers {
         githubPullRequest {
+            useGitHubHooks()
+            userWhitelist(["appformer", "kiegroup"])
```
Comment (mareknovotny) [inline job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: userWhiteList? I thought that orgWhitelist is enough?

File: job-dsls/jobs/pr_jobs.groovy [hunk 2]
```diff
@@ -216,13 +216,14 @@ for (repoConfig in REPO_CONFIGS) {
 
         triggers {
             githubPullRequest {
+                useGitHubHooks()
                 orgWhitelist(["kiegroup"])
+                userWhitelist(["kiegroup"])
```
Comment (mareknovotny) [inline job-dsls/jobs/pr_jobs.groovy:221]: userWhiteList? I thought that orgWhitelist is enough?

File: job-dsls/jobs/springboot_pr_job.groovy [hunk 3]
```diff
@@ -97,15 +97,16 @@ job(jobName) {
 
     triggers {
         githubPullRequest {
+            useGitHubHooks()
+            userWhitelist(["appformer", "kiegroup"])
```
Comment (mareknovotny) [inline job-dsls/jobs/springboot_pr_job.groovy:101]: userWhiteList? I thought that orgWhitelist is enough?

File: job-dsls/jobs/kie_jenkinsScripts_PR.groovy (hunk 1 above)
Comment (Ginxo) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: @mareknovotny we tested it and for some reason it is required by the hook. It didn't work without it. I would like to know the exact reason why it's happening but I don't, I will try to investigate it and will let you know as soon as I have something.

Comment (mareknovotny) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: ok

Comment (mareknovotny) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: please add there a comment about this why it is there as we could forget and remove it accidentally

Comment (Ginxo) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: @mareknovotny let me make a final test once jenkins is back just to be sure I'm not wrong with this

Comment (Ginxo) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:88]: So I was confused, the userWhiteList is not needed

File: job-dsls/jobs/pr_jobs.groovy (hunk 2 above)
Comment (Ginxo) [reply job-dsls/jobs/pr_jobs.groovy]: removed

File: job-dsls/jobs/kie_jenkinsScripts_PR.groovy (hunk 1 above)
Comment (Ginxo) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy]: removed

**Pull request merged**